torchdrivesim/simulator.py

```diff
@@ -420,7 +427,9 @@ def _compute_collision_of_single_agent(self, box: Tensor, remove_self_overlap: b
 
         Args:
             box: Bx5 tensor, with the last dimension being (x,y,length,width,psi).
-            remove_self_overlap: if the input agent is present in the simulation, set this to subtract self-overlap
+            remove_self_overlap: B boolean tensor, where if the input agent is present in the simulation,
+                set this to subtract self-overlap. By default it is assumed that self overlapping exists and will be removed.
+            agent_types: An optional list of specific agent types for computing collisions with
```

Comment:
Explain what happens when this list is not provided.

Reply:
Done.